esalazarv/oyster-test
# Add User CRUD

[added]
- user resource
- bcrypt
- validation pipe
- confirm decorator
- e2e test of main endpoints (list and create)
- add user migrations
- seed admin user

[to do]
- add missing e2e test for user controller
